en/docs/learn/analytics/analyzing-apim-statistics-with-batch-analytics/using-geo-location-based-statistics/configuring-geolocation-based-statistics.md

```diff
@@ -6,48 +6,84 @@
+    ``` sql
+    load data local infile '[PATH_TO_GeoLite2-City-Locations-en]/GeoLite2-City-Locations-en.csv' into table LOCATION
+    fields terminated by ','
+    enclosed by '"'
+    lines terminated by '\n'
+    (geoname_id, locale_code, continent_code, continent_name, country_iso_code, country_name, subdivision_1_iso_code, subdivision_1_name, subdivision_2_iso_code, subdivision_2_name, city_name, metro_code, time_zone);
+    ```
 
-    - Importing **Geolocation Data/data/BLOCKS.csv**
+    !!! tip
+        If you get an error when importing data from `GeoLite2-City-Locations-en.csv` to `LOCATION` table in PostgreSQL(as `GeoLite2-City-Locations-en.csv` file may include additional columns like 'is_in_european_union' which are not required) you can alter the `LOCATION` table to include non required columns and continue with the importing. Once the importing is successful, you can drop those non required columns.      
+
+5.  Check whether your imported dataset is working properly by executing following query in the MySQL Command Line.
+    ``` sql
+    SELECT loc.country_name,loc.subdivision_1_name
+    FROM BLOCKS block , LOCATION loc
+    WHERE block.network_blocks = '<Network_part_of_ipv4>'
+    AND <Long_value_of_publilc_IP> BETWEEN block.network
+    AND block.broadcast AND block.geoname_id=loc.geoname_id;
+    ```
 
-        `mysqlimport -u root -p --ignore-lines=2 --fields-terminated-by=, --fields-optionally-enclosed-by='"' --local GEO_LOCATION_DATA <Extracted_location>/GeolocationData/data/BLOCKS.csv`
+    **Example query** :
 
-        !!! tip
-            For more information, see [MySQL Documentation - Data Export and Import](https://dev.mysql.com/doc/workbench/en/wb-admin-export-import.html).
+    ``` sql
+    SELECT loc.country_name,loc.subdivision_1_name
+    FROM BLOCKS block , LOCATION loc
+    WHERE block.network_blocks = '221.192'
+    AND 3720398641 BETWEEN block.network
+    AND block.broadcast AND block.geoname_id=loc.geoname_id;
+    ```
 
-4.  Check whether your imported dataset is working properly by executing following query in the MySQL Command Line.
+6.  Download a JDBC provider depending on the database you are using (MySQL, in this example), and copy it to the `<API-M_ANALYTICS_HOME>/repository/components/lib` directory.
 
-        SELECT loc.country_name,loc.subdivision_1_name
-        FROM BLOCKS block , LOCATION loc
-        WHERE block.network_blocks = '<Network_part_of_ipv4>'
-        AND <Long_value_of_publilc_IP> BETWEEN block.network
-        AND block.broadcast AND block.geoname_id=loc.geoname_id;
+    !!! info
+        If the JDBC driver is not an OSGI bundle, then it should be converted to OSGI (using jartobundle.sh) before placing it in the
+        `<API-M_ANALYTICS_HOME>/lib` directory. For detailed instructions, see [Adding Third Party Non OSGi Libraries](../../../adding-third-party-non-osgi-libraries/).
```

Comment:
Check this link. It should start with {{base_path}}

Reply:
Fixed in d9c29843ef4201cb9eca588053bea4aaecd174e0